packages/integration_test/lib/integration_test_driver.dart

```diff
@@ -8,11 +8,11 @@ import 'dart:io';
 
 import 'package:flutter_driver/flutter_driver.dart';
 
-import 'package:e2e/common.dart' as e2e;
+import 'package:integration_test/common.dart';
 import 'package:path/path.dart' as path;
 
 /// This method remains for backword compatibility.
-Future<void> main() => e2eDriver();
+Future<void> integration_main() => integrationDriver();
```

Comment:
If we are not calling it `main` anyway, why not just remove it? I was thinking if it's `main` it automatically can be used as a driver script.